docs/source/theano.rst

```diff
@@ -0,0 +1,157 @@
+# pymc3 and theano
```

Comment:
I thought it looked strange in the text, but somehow I didn't think about that :-)